cirq/sim/wave_function.py

```diff
@@ -93,13 +94,16 @@ def density_matrix_of(self, qubits: List[ops.Qid] = None) -> np.ndarray:
         standard Kronecker convention of numpy.kron.
 
         For example:
-            self.state_vector() = np.array([1/np.sqrt(2), 1/np.sqrt(2)],
-                dtype=np.complex64)
-            qubits = None
-            gives us \rho = \begin{bmatrix}
-                                0.5 & 0.5
-                                0.5 & 0.5
-                            \end{bmatrix}
+        self.state_vector() = np.array([1/np.sqrt(2), 1/np.sqrt(2)],
```

Comment:
can this be indented if there is a newline between "For example" and the code?

Reply:
Unfortunately no, if I indent this it somehow creates a warning, `WARNING: Field list ends without a blank line; unexpected unindent.`, even though it renders just fine.

Reply:
What happens if you put that command in one line?
 ```
For example:
            self.state_vector() = np.array([1/np.sqrt(2), 1/np.sqrt(2)], dtype=np.complex64)
```

Reply:
It renders the same as no indentation.

Reply:
Does it give a warning?

Reply:
@mpharrigan This one ^ left :)

Reply:
@vtomole No, it doesn't give a warning. However to get any change in the generated doc, it needs to be a newline as suggested by @dabacon. But that causes some interference with the indented math further down, giving rise to the original warning. 

This is how things looked before:
![image](https://user-images.githubusercontent.com/23558186/63505473-e7f13900-c4d3-11e9-91b7-6630d9cd7417.png)
![image](https://user-images.githubusercontent.com/23558186/63505520-f5a6be80-c4d3-11e9-8462-890d57cf2b3b.png)

To which I prefer the new looks:
![image](https://user-images.githubusercontent.com/23558186/63505703-4ae2d000-c4d4-11e9-8ae8-fc5ef708b554.png)
![image](https://user-images.githubusercontent.com/23558186/63505717-52a27480-c4d4-11e9-8467-6f2a80087100.png)

It would be nice to get the code examples in their own boxes. What @dabacon  suggested yields:
![image](https://user-images.githubusercontent.com/23558186/63506287-8500a180-c4d5-11e9-9f02-abd74c10c1c6.png)
Which looks slightly nicer. However, that generates warnings...